api/src/utils/mailer.ts

```diff
@@ -0,0 +1,65 @@
+import nodemailer from 'nodemailer';
+import { IUser, IPitch } from 'ssw-common';
+import { pitchStatusEnum } from './enums';
+
+type message = { to: string; from: string; subject: string; html: string };
```

Comment:
Types should be capitalized

Reply:
This also feels like it should be more of an Interface than a type because the type is just can object.

```typescript
interface EmailMessage {
to: string;
from: string;
subject: string;
html: string
}
```